interfaces/builtin/utils.go

```diff
@@ -102,3 +104,19 @@ func implicitSystemConnectedSlot(slot *interfaces.ConnectedSlot) bool {
 	}
 	return false
 }
+
+// determine if the given slot attribute path matches the regex
+func verifySlotPathAttribute(slotRef *interfaces.SlotRef, attrs interfaces.Attrer, reg *regexp.Regexp, errStr string) (string, error) {
```

Comment:
s/errStr/attrMustBeMsg/ , also  the doc comment should probably explain an example of the resulting full error, otherwise people might pass in strings that make for non-grammatical errors